Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc

```diff
@@ -1,8 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-#ifndef HOLOTOOLKIT_COMMON
-#define HOLOTOOLKIT_COMMON
+#ifndef MIXEDREALITYTOOLKIT_COMMON
+#define MIXEDREALITYTOOLKIT_COMMON
+
+#include "UnityCG.cginc"
```

Comment:
Do we need this new `#include`, though we didn't have it before?

Reply:
Strangely enough, Yes. Wouldn't compile without it.  Very odd.

Reply:
Huh, strange!